pom.xml

```diff
@@ -70,7 +70,7 @@
     <javax.annotations.version>1.3.2</javax.annotations.version>
     <animal-sniffer.version>1.19</animal-sniffer.version>
     <iam.version>1.0.7</iam.version>
-    <opencensus.version>0.24.0</opencensus.version>
+    <opencensus.version>0.28.0</opencensus.version>
```

Comment:
We were trying to avoid upgrading this one.

Reply:
It looks like there are no breaking changes here and we should consider at least 0.28.1 which supposedly adds [support for OpenTelemetry migration](https://github.com/census-instrumentation/opencensus-java/pull/2059)

Reply:
Done.